scripts/legislation/africa/zanzibar_english.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^ENACTED by the House of Representatives of Zanzibar/,$p' | \
+    sed -n '/^1/,$p' 
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/\(([A-Za-z0-9]+)\)/[\1]/g'
+}
+
+function amend_errors_in_articles {
+  sed -E 's/ 1\]/ [1]/g' | \
+    sed -E 's/ –/:/g' | \
+    sed -E "s/‘([A-z]+|[A-z]+ [A-z]+|[A-z]+ [A-z]+ [A-z]+)'/|\1|/g" | \
+    sed -E 's/\|([A-z]+|[A-z]+ [A-z]+|[A-z]+ [A-z]+ [A-z]+)\|/"\1"/g' | \
```

Comment:
These 2 lines are intended to replace incorrect quotation marks primarily appearing in Article 2 such as `‘air'` with the correct quotation marks. Because I could not include both `'` and `"` in one line I used the `|` initially and then replaced it with the double quotation mark. The use of multiple `[A-z]+` is intended to capture multiple words which are between the incorrect quotation marks.

Reply:
Oh, I see. Thanks for the clarification.